holdings_maintenance/pds4/pds4infoshelf.py

```
@@ -0,0 +1,896 @@
+        pdsf = pdsfile.Pds4File.from_abspath(path)
+
+        if pdsf.is_bundleset_dir:
+            # Info about archive directories is stored by bundleset
+            if pdsf.archives_:
+                info.append((pdsf, None))
+
+            # Others are checksumed by bundle
+            else:
+                children = [pdsf.child(c) for c in pdsf.childnames]
+                info += [(c, None) for c in children if c.isdir]
+                        # "if c.isdir" is False for bundleset level readme files
```

Flake8 (7.2.0)

[error] unexpected indentation (comment)

(E116)